### ISSUE TITLE
Documentation notice on the size mixin

### ISSUE THREAD

**Issue by Xmetalfanx** [thoughtbot/bourbon#1011]
Hate to put this here, .. its not a big deal but something I noticed on the Docs 

### Expected behavior
CSS to output as expected 

Following http://bourbon.io/docs/#size for the size mixin 

### Actual behavior
Got error with Sass Watch   
`error sass/bourbon/bourbon/library/_size.scss (Line 41: `98% 25px` is not a valid length for the `$height` argument in the `size` mixin.)`

Sass being:         @include size(98% 25px);

(values are not important... just an example) 

## "Fix"
basically there is just a comma missing in the documentation between the values.   Nothing that was hard for me to figure out, but some people may be scratching their heads running into this error, following the documentation, EXACTLY.   

In my example here the corrected Sass is         `@include size(98%, 25px); `

**Comment by tysongach** [comment]
@Xmetalfanx Are you using the 5.0 beta? If so, bourbon.io is currently the documentation for the 4.x release, not 5.0. 

Once we release 5.0 out of beta, the website will be updated.

**Comment by Xmetalfanx** [comment]
Looks like I am using a beta version and the documentation just hasn't been updated yet.
